java/vector/pom.xml

```diff
@@ -133,6 +134,37 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+       <groupId>org.apache.maven.plugins</groupId>
+       <artifactId>maven-shade-plugin</artifactId>
+       <version>3.2.1</version>
```

Comment:
Can we stick to the same version of shade plugin? it looks like 3.1.1 in flight

Reply:
yes.